### PR TITLE
Frontend release pages

### PR DESCRIPTION
**Pull request by phlpphcknr**
- set up pages to display releases for each artist in FE
- minor adaptions

### CODE REVIEW

File: frontend/src/pages/ArtistReleaseOverview.js [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+            <section>
+                Loading
+            </section>
+        )
+    }
+
+    function releaseList () {
+        if(releaseType === "albums"){
+            return (
+                <>
```
Comment (christophersiem) [inline frontend/src/pages/ArtistReleaseOverview.js:30]: don't need fragments here

File: frontend/src/pages/ArtistReleaseOverview.js [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+                    {artist.artistAlbums.slice(0, 5).map((release) =>
+                        <ReleaseDescription release={release}/>)}
+                </>
+            )
+        }else if (releaseType === "singles-EPs"){
+            return (
+                <>
```
Comment (christophersiem) [inline frontend/src/pages/ArtistReleaseOverview.js:37]: don't need fragments here

File: frontend/src/pages/ArtistReleaseOverview.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+        </ReleaseOverview>
+    )
+
+}
+
+
+
+
+/*{artist.artistAlbums.slice(0, 5).map((release) =>
+    <ReleaseDescription release={release}/>)}*/
```
Comment (christophersiem) [inline frontend/src/pages/ArtistReleaseOverview.js:62]:
```suggestion

```

File: frontend/src/pages/ArtistReleaseOverview.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
+import {Link, useParams} from "react-router-dom";
+import {useEffect, useState} from "react";
+import {getArtistByName} from "../service/musiComApiService";
+import styled from "styled-components/macro";
+import ArtistHeader from "../components/ArtistHeader";
+import ReleaseDescription from "../components/ReleaseDescription";
+
+export default function ArtistReleaseOverview(){
+    const {artistName} = useParams();
+    const {releaseType} = useParams();
```
Comment (christophersiem) [inline frontend/src/pages/ArtistReleaseOverview.js:10]:
```suggestion
    const {artistName, releaseType} = useParams();
```

File: frontend/src/pages/ArtistSearchResult.js [hunk 5]
```diff
@@ -33,7 +35,7 @@ const SearchResults = styled.div`
   p {
     margin-bottom: 15px;
     margin-top: 15px;
```
Comment (christophersiem) [inline frontend/src/pages/ArtistSearchResult.js:37]:
```suggestion
    margin: 15px 0px

```

**Review by christophersiem — APPROVED** [review]
lgtm 👍🏽

File: frontend/src/App.js [hunk 6]
```diff
@@ -30,4 +36,12 @@ const PageLayout = styled.div`
   overflow-y: scroll;
   grid-template-rows: auto 1fr;
   background: var(--secondary-color);
+  
+  AppHeader{
+    
+  }
```
[the remark this answers is not in the document]
Comment (phlpphcknr) [reply frontend/src/App.js:42]: thx. done

File: frontend/src/components/AppHeader.js [hunk 7]
```diff
@@ -14,14 +14,16 @@ const Header = styled.header`
   box-shadow: 0px 2px 8px #333;
   padding: 10px 0px 20px 0px;
```
[the remark this answers is not in the document]
Comment (phlpphcknr) [reply frontend/src/components/AppHeader.js:15]: thx

File: frontend/src/components/ArtistHeader.js [hunk 8]
```diff
@@ -3,12 +3,17 @@ import {Link} from "react-router-dom";
 
 export default function ArtistHeader({artist}) {
 
-
+    function artistImage () {
+        if(artist.artistInfo.artistImageUrl === ""){
+            return <img src={"https://upload.wikimedia.org/wikipedia/commons/8/89/Portrait_Placeholder.png"} alt="" />;
+        }
+        return <img src={artist.artistInfo.artistImageUrl} alt="" />;
```
[the remark this answers is not in the document]
Comment (phlpphcknr) [reply frontend/src/components/ArtistHeader.js:10]: ok. done

File: frontend/src/pages/ArtistReleaseOverview.js (hunk 1 above)
Comment (phlpphcknr) [reply frontend/src/pages/ArtistReleaseOverview.js:30]: alright, shortens the code a little ...

File: frontend/src/pages/ArtistReleaseOverview.js (hunk 3 above)
Comment (phlpphcknr) [reply frontend/src/pages/ArtistReleaseOverview.js:62]: thx, now deleted...

File: frontend/src/pages/ArtistReleaseOverview.js (hunk 4 above)
Comment (phlpphcknr) [reply frontend/src/pages/ArtistReleaseOverview.js:10]: thx, nice...

File: frontend/src/pages/ArtistReleaseOverview.js (hunk 2 above)
Comment (phlpphcknr) [reply frontend/src/pages/ArtistReleaseOverview.js:37]: if I delete them I get a compiler error ...

File: frontend/src/pages/ArtistReleaseOverview.js (hunk 1 above)
Comment (phlpphcknr) [reply frontend/src/pages/ArtistReleaseOverview.js:30]: I saw now that I get a compiler error when I delete them ...

File: frontend/src/pages/ArtistSearchResult.js (hunk 5 above)
Comment (phlpphcknr) [reply frontend/src/pages/ArtistSearchResult.js:37]: thx. done

**Pull request merged**